grails-app/services/org/pih/warehouse/receiving/ReceiptService.groovy

```diff
@@ -528,4 +528,14 @@ class ReceiptService {
 
         saveAndCompletePartialReceipt(partialReceipt)
     }
+
+    Location findReceiveLocation(Shipment shipment) {
+        def inboundSortationLocation = Location.findByName(Constants.INBOUND_SORTATION_LOCATION_NAME)
```

Comment:
I think I'd rather rely on the activity codes instead location name (needs to have receive stock, not have pick stock, anything else?). On top of that, you have to look for internal locations from a destination, because when doing a simple `findBy`, you can get internal locations from different depots that you are receiving into.